client/store.js

```diff
@@ -244,6 +254,12 @@ store.setupKresus = function(cb) {
         has(world, 'accounts');
         has(world, 'operations');
 
+        let defaultAccountId = store.getDefaultAccountId();
+        if (defaultAccountId && !data.currentAccountId)
+        {
+            data.currentAccountId = defaultAccountId;
```

Comment:
Can you:
1. Move this after accounts have been set
2. Assert that the defaultAccountId value actually matches one of the accounts ids?